packages/transport-native/android/src/main/java/io/trezor/transport/TrezorTransportModule.java

```diff
@@ -0,0 +1,132 @@
+package io.trezor.transport;
+
+import android.util.Log;
+
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.WritableArray;
+import com.facebook.react.bridge.WritableMap;
+
+import io.trezor.transport.bridges.UDPBridge;
+import io.trezor.transport.bridges.USBBridge;
+import io.trezor.transport.interfaces.BridgeInterface;
+import io.trezor.transport.interfaces.TrezorInterface;
+
+import java.util.List;
+
+public class TrezorTransportModule extends ReactContextBaseJavaModule {
+  private static final String TAG = "Trezor BridgeModule";
+  private static ReactApplicationContext reactContext;
+  private static BridgeInterface bridge;
+
+
+  public TrezorTransportModule(ReactApplicationContext context) {
+    super(context);
+    reactContext = context;
+
+    // TODO: figure out smarter way ho to turn emulator on
```

Comment:
is this TODO comment belongs here? looks like some leftover

Reply:
removed